.github/workflows/ci-tests.yml

```diff
@@ -35,9 +35,13 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest"]
-        python-version: ["3.10"]
+        python-version: ["3.11"]
         session: ["doctest", "gallery", "linkcheck"]
         include:
+          - os: "ubuntu-latest"
+            python-version: "3.11"
+            session: "tests"
+            coverage: "--coverage"
           - os: "ubuntu-latest"
             python-version: "3.10"
             session: "tests"
```

Comment:
@rcomer We should only really be submitting test coverage for one version of `python`, so remove the `coverage: "--coverage"` for `python-version: "3.10"` on `line +48` in the `matrix` :+1: